tests/tensorflow/test_printing.py

```diff
@@ -94,16 +94,26 @@ def test_shape_index():
 
     z = tf.matmul(A, tf.add(y, y, name='x_p_y'), name='A_dot')
 
+    
     range_out = io.StringIO()
     with redirect_stdout(range_out):        
         tf_dprint(z, depth_index=range(3, 6))
 
     expected_out = textwrap.dedent('''
+    .
+    .
+    .
     |  |  |  Op(Add)\t"x_p_y"
     |  |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
     |  |  |  |  |  Op(Mul)\t"y"
+    .
+    .
+    .
     |  |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
     |  |  |  |  |  Op(Mul)\t"y"
+    .
+    .
+    .
```

Comment:
The purpose of this indexing functionality is to make large graphs viewable&mdash;in part by restricting the output to only certain subgraphs&mdash;but this new form of output doesn't accomplish that.  

Before this change, a large graph would push all the relevant content to the right an amount of pipes equal to the graph's lower-indexed depth, and now, with this change, it will **also** flood the screen vertically.  The end result is a negligible improvement over the non-indexed output.

More helpful truncated/indexed output would show **only the indexed subgraph** (i.e. without any of the preceding depth indicators/pipes) and a simple indication of the depth(s) of said subgraph.

For example:
```
3 <= depth <= 5
... Op(Add)  "x_p_y"
... |  Tensor(Mul):0,  shape=[None, 1]  "y:0"
... |  |  Op(Mul)  "y"
... |  |  | ...
... |  Tensor(Mul):0,  shape=[None, 1]  "z:0"
... |  |  Op(Mul)  "z"
... |  |  | ...
```

Reply:
I removed the original notation for truncating and made it resemble the example above.